src/Compilers/Test/Utilities/CSharp/Extensions.cs

```diff
@@ -554,4 +554,29 @@ internal static Symbol GetDeclaredSymbolFromSyntaxNode(this CSharpSemanticModel
 
         return (Symbol)model.GetDeclaredSymbol(declaration, cancellationToken);
     }
+
+    public static NamedTypeSymbol BaseType(this TypeSymbol symbol)
```

Comment:
I'd be tempted to remove those extensions and update the tests to call the new property/method names instead. But it's not essential.